Mathlib/SetTheory/ZFC/Basic.lean

```diff
@@ -811,6 +827,23 @@ theorem toSet_inj {x y : ZFSet} : x.toSet = y.toSet ↔ x = y :=
 instance : IsAntisymm ZFSet (· ⊆ ·) :=
   ⟨fun _ _ hab hba => ext fun c => ⟨@hab c, @hba c⟩⟩
 
+instance : PartialOrder ZFSet where
+  le := (· ⊆ ·)
+  le_refl := refl_of (· ⊆ ·)
+  le_trans _ _ _ := trans_of (· ⊆ ·)
+  le_antisymm _ _ := antisymm_of (· ⊆ ·)
+
+instance : HasSSubset ZFSet :=
+  ⟨(· < ·)⟩
+
+@[simp]
+theorem le_def (x y : ZFSet) : x ≤ y ↔ x ⊆ y :=
+  Iff.rfl
+
+@[simp]
+theorem lt_def (x y : ZFSet) : x < y ↔ x ⊂ y :=
```

Comment:
With this setup we actually don't have much proving things about ssubset on ZFSet and PSet, even though it's the simp normal form. Probably the easiest way to prove something about it with the current would be to use this lemma in reverse, then use a lemma relating < to <=, then using le_def and then something cleaner can happen... Instead, we should have some API relating ssubset with mem directly. To not clog up this PR, let's have a todo on the HasSSubset instance recording this.